Gemfile

```
@@ -0,0 +1,62 @@
+source 'https://rubygems.org'
+git_source(:github) { |repo| "https://github.com/#{repo}.git" }
+
+ruby '2.5.3'
+
+# Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
+gem 'rails', '~> 5.2.2'
+# Use sqlite3 as the database for Active Record
+gem 'sqlite3'
```

thay dấu nháy đơn thành nháy kép.
```suggestion
gem "sqlite3"
```